docs/getting-started/installation/install-redis-from-source.md

```diff
@@ -18,6 +18,12 @@ To obtain the source files for the latest stable version of Redis from the Redis
 wget https://download.redis.io/redis-stable.tar.gz
 {{< / highlight >}}
 
+## Install needed dependencies
+
+{{< highlight bash >}}
+apt install gcc -y
```

Comment:
I usually do `apt install build-essential tcl8.5` - this tales care of most deps and also prepares for running `make test`.